scripts/india_wris/base.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pandas as pd
+import json
+import re
+from india.geo.districts import IndiaDistrictsMapper
+
+
+class WaterQualityBase():
+
+    def __init__(self, dataset_name, util_names, template_strings):
+        self.dataset_name = dataset_name
+        self.util_names = util_names
+        self.module_dir = os.path.dirname(__file__)
+
+        self.solute_mcf = template_strings['solute_mcf']
```

Comment:
Nit: add assertions that all the required properties of template_strings are available?

Reply:
Added assert statements in the `base.py` file. Please check if it's fine and readable.

Reply:
this looks good!